.github/workflows/p2p-workflow-fastfeedback.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
   promote:
     name: promote
     needs: [nft-test, functional-test]
-    if: success() && github.ref == inputs.main-branch
+    if: success() && (github.ref == inputs.main-branch || startsWith(github.ref,  'refs/tags/'))
```

Comment:
Did you try something like `github.ref_type == 'tag'`?

```
github.ref_type
string
The type of ref that triggered the workflow run. Valid values are branch or tag.
```

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
that's nicer actually, Ill switch